src/main/java/at/hannibal2/skyhanni/utils/renderables/Renderable.kt

```diff
@@ -761,14 +723,14 @@ interface Renderable {
                     content.render(posX, posY)
                 } else if (!shouldRenderTopElseBottom) {
                     content.render(posX, posY)
-                    GlStateManager.translate(0f, (ySpacing).toFloat(), 0f)
+                    DrawContextUtils.translate(0f, (ySpacing).toFloat(), 0f)
```

Comment:
the replace of gl state mánager to draw context utils calls is not relevant ot the container renderable change, again please try to avoid unnecessary changes in bigger prs in the future

Reply:
Cal asked me to.
https://github.com/hannibal002/SkyHanni/pull/3890#pullrequestreview-2795803768